dali/operators/geometry/affine_transforms/transform_crop.cc

```diff
@@ -17,7 +17,7 @@
 
 namespace dali {
 
-DALI_SCHEMA(TransformCrop)
+DALI_SCHEMA(transforms__Crop)
```

Comment:
I wonder if we should use `__` as a separator. Maybe we can add a separate entry in schema regarding module or use some special character as a separator, like `#` or `$`?

Reply:
This would take extensive changes in the registries and registration macros - now the name is a part of a C++ identifier, and these must consist solely of alphanumeric signs.

Reply:
I tried to use a different name for registration, but then we had to supply two of them - one for C++, the other for the registry. Not ideal.

Reply:
ok

Reply:
Just checked, '$' works OK. Do we prefer it over '__' ?

Reply:
@mzient ?
It is also a part of C++ identifier, so it may look strange with `$` in the middle.

Reply:
It works in GCC and MSVC, but it's not listed in ISO standard as a valid character in identifiers.